jenkins/ring-install.sh

```diff
@@ -443,8 +443,15 @@ function purge_ring {
 function test_sproxyd {
     for path in 'chord_path' 'arc'; do
         local r=$RANDOM
+        local put_response=${r}_put
+        local get_response=${r}_get
         local url="http://localhost:81/proxy/${path}/${r}"
-        curl -v -XPUT -H "Expect:" -H "x-scal-usermd: bXl1c2VybWQ=" ${url} --data-binary @/etc/hosts
-        curl -v -XGET ${url}
+        curl -i -XPUT -H "Expect:" -H "x-scal-usermd: bXl1c2VybWQ=" ${url} --data-binary @/etc/hosts -o $put_response
+        cat $put_response
```

Comment:
Do get more information on what went wrong in case of a failure, with the pipe solution we get no output.